### DISCUSSION TITLE
Parse Errors - App Failed to Compile

### DISCUSSION THREAD

**Discussion by cmeffley** [nss-evening-cohort-14/evening-client-side#339] [Capstones!] [answered]
Chris Meffley

other ticket: #32

I npm installed recharts for a stretch goal, and started getting Failed to compile with this error:
```
src\views\AveragesView.js
  Line 4:29:  Parse errors in imported module '../components/WorkoutAverages': Line 53: Unexpected token

  51 |             <CartesianGrid strokeDasharray="3 3" />
  52 |             <XAxis dataKey=''
> 53 |           </LineChart>
     |           ^
  54 |         </ResponsiveContainer>
  55 |       </div>
  56 |       <Card className='datacards'> (53:11)  import/no-named-as-default
  Line 4:29:  Parse errors in imported module '../components/WorkoutAverages': Line 53: Unexpected token
```

https://github.com/recharts/recharts/issues/1057
https://stackoverflow.com/questions/36228249/eslint-throws-parsing-error-unexpected-token-on-named-export
https://stackoverflow.com/questions/36001552/eslint-parsing-error-unexpected-token

Tried:
- Some sugguestions were to add 'babel-eslint' in the .eslintrc file, which I already have.
- Deleted all the code for the charts and uninstalled Recharts, but it's still giving the failed to compile parse error.
- I was working on a branch, but when I switched to my development branch and start the server the error still happens.

File:
https://github.com/cmeffley/front-end-capstone/blob/main/src/components/WorkoutAverages.js

**Comment by drteresavasquez** [comment, marked as answer]
@cmeffley if this is a recurring issue after uninstalling the package, I would not use it. If your code is good on Github, since your MVP branch should be ready make sure after the uninstall, you remove node modules, do another npm i and restart your server after. Hopefully that gets the job done.

**Reply by cmeffley** [reply]
It worked!! Thank you!
